.env.example

```diff
@@ -3,8 +3,8 @@ DB_PASSWORD=db_password
 DB_NAME=barefoot_nomad
 DB_HOST=127.0.0.1
 DB_TEST_NAME=barefoot_test
-PORT = 
-EMAIL = 
-LAYER = Nodemailer package require you provide the LAYER which is the sender mailling service provider. It can be e.g include Yahoo, gmail and your private account mail provider as well.we intend
-PASSWORD = 
-SECRET_KEY_EMAIL_VERIFY_TOKEN = 
+PORT =  
+NODE_ENV=test
+SALT=
```

Comment:
Why did you remove the other env variables?. You should just update it with yours

Reply:
ok

Reply:
I only resolved the conflict since it has not been merged